### ISSUE TITLE
No thumbs up?

### ISSUE THREAD

**Issue by matthewwithanm** [jsumners/alfred-emoji#2]
When I search for "thumb," neither thumbs up nor thumbs down comes up.

**Comment by jsumners** [comment]
Typing `+1` works for me. This workflow uses the aliases as defined in the `emojilib` package.

**Comment by jsumners** [comment]
Hmmm, `emojilib` has the following definitions:

```
"+1": {
    "keywords": ["thumbsup", "yes", "awesome", "good", "agree", "accept", "cool", "hand", "like"],
    "char": "👍",
    "fitzpatrick_scale": true,
    "category": "people"
  },
  "-1": {
    "keywords": ["thumbsdown", "no", "dislike", "hand"],
    "char": "👎",
    "fitzpatrick_scale": true,
    "category": "people"
  }
```

None of the keywords are matching for either of these characters.

**Comment by jsumners** [comment]
`puke` is also failing to return a result.

**Comment by jsumners** [comment]
Fixed in v1.0.1
